app/services/session_encryptor.rb

```diff
@@ -8,15 +8,13 @@ def self.build_user_access_key
     build_user_access_key
   end
 
-  # rubocop:disable Security/MarshalLoad
   def self.load(value)
     decrypted = encryptor.decrypt(value, user_access_key)
-    Marshal.load(::Base64.decode64(decrypted))
+    ActiveSupport::JSON.decode(::Base64.decode64(decrypted))
```

Comment:
Benefits to `with_indifferent_access`:

- symbol keys are *very* common in Ruby, I believe it matches developer expectations
- makes the diff for this PR much smaller (and less risky)

Cons:
- speed? We've had this conversation before, but IMO we don't use Ruby for its speed. Rails is already making hashes with indifferent access for params in every single request, so I don't think another hash is going to make a huge difference here.